sdk/kiosk/src/tx/kiosk.ts

```diff
@@ -200,20 +203,16 @@ export function purchase(
   tx: TransactionBlock,
```

Comment:
Shall we call txb to align with other places?

Reply:
I think in this sdk, we have it defined as `tx` everywhere!
Does `other places` refer to the `@mysten/sui.js` sdk?